acapy_agent/wallet/default_verification_key_strategy.py

```diff
@@ -55,10 +88,50 @@ async def get_verification_method_id_for_did(
         :params proof_purpose: the verkey relationship (assertionMethod, keyAgreement, ..)
         :returns Optional[str]: the current verkey ID
         """
+        proof_type = proof_type or "Ed25519Signature2018"
```

Comment:
I would like to suggest a slight nudge towards `Ed25519Signature2020` as default option simply to hint at driving the community forward. `Ed25519Signature2018` is a thing of the past.
```suggestion
        proof_type = proof_type or "Ed25519Signature2020"
```

Reply:
I don't disagree that we should be heading towards bigger and better; this default was motivated by backwards compatibility. I don't know how much of an impact it would have to change the default so I'm reluctant to do so until that's well known.

Reply:
That's fair. This is not likely a feature I will be using so I'm not too opinionated about it.